src/Controller/PublishersController.php

```diff
@@ -104,4 +104,21 @@ public function edit(Request $request, int $id): Response
             'publisher_form' => $form->createView()
         ]);
     }
+
+    /**
+     * @Route ("/deletePublisher/{id}", name="app_deletePublisher")
+     */
+    public function delete(Publisher $publisher): Response
+    {
+        if ($publisher->getGames()->isEmpty()) {
+            $response = $this->dataFactory->deletePublisher($publisher);
+            $type = "success";
```

Comment:
Make these type variables into constants in this class;
``
const SUCCESS_TYPE = "success";
``
And make sure to use those constants everywhere, instead of re-writing same strings.